File: PRAK501-2310817210001-AndraBraputraAkbarSaleh.py
def max(a, b, c, d):
    nilai = a if a > b else b
    nilai = c if c > nilai else nilai
    nilai = d if d > nilai else nilai
    return nilai

def main():
    a, b, c, d = map(int, input("").split())
    hasil = max(a, b, c, d)
    print(hasil)

if __name__ == "__main__":
    main()

File: PRAK501-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>
int Max(int a, int b, int c, int d) {
    int a2 = (a > b) ? a : b;
    int b2 = (c > d) ? c : d;
    return (a2 > b2) ? a2 : b2;
}
int main() {
int a, b, c, d; 
scanf("%d %d %d %d", &a, &b, &c, &d);
int hasil = Max(a, b, c, d);
printf("%d", hasil); 
return 0;
}
